frontend/src/console/components/AddDomainButton/AddDomainButton.tsx

```diff
@@ -0,0 +1,269 @@
+              <Row>
+                <Col lg={8}>
+                  <TextInputField
+                    value={this.state.externalDomain}
+                    onChange={this.handleChange}
+                    type="domain"
+                    messages={messages}
+                    fieldName="externalDomain"
+                    autoComplete="off"
+                  />
+                  {instance.feedback.externalDomain ? (
+                    <div className="domain-error">
+                      <p className="error-msg">
+                        <i className="fas fa-exclamation-triangle pr-2" />
+                        <WrappedMessage
+                          messages={messages}
+                          id="domainErrorMessage"
+                        />
+                      </p>
+                      <p className="error-detail">
+                        <WrappedMessage
+                          messages={messages}
+                          id="domainErrorDescription1"
+                        />
+                        <a href={SUPPORT_LINK}>contact support</a>
+                        <WrappedMessage
+                          messages={messages}
+                          id="domainErrorDescription2"
+                        />
+                      </p>
+                    </div>
+                  ) : null}
+                </Col>
+              </Row>
+              <Form.Group>
+                <Form.Check type="checkbox" className="d-flex my-3">
+                  <Form.Check.Input
+                    type="checkbox"
+                    checked={this.state.domainRightsAggrement}
+                    onChange={this.toggleDomainRights}
+                  />
+                  <Form.Check.Label className="checkbox-label">
+                    <WrappedMessage
+                      messages={messages}
+                      id="domainRightsAgreement"
+                    />
+                  </Form.Check.Label>
+                </Form.Check>
+                <Form.Check type="checkbox" className="d-flex my-3">
+                  <Form.Check.Input
+                    type="checkbox"
+                    checked={this.state.additionalFeeAggrement}
+                    onChange={this.toggleAdditionalFee}
+                  />
+                  <Form.Check.Label className="checkbox-label">
+                    <WrappedMessage
+                      messages={messages}
+                      id="additionalFeeAgreement"
+                    />
+                  </Form.Check.Label>
+                </Form.Check>
+              </Form.Group>
+              <div className="d-flex flex-row">
+                <div className="verify-btn">
+                  <Button
+                    size="lg"
+                    variant="primary"
+                    disabled={
+                      !(
+                        this.state.additionalFeeAggrement &&
+                        this.state.domainRightsAggrement
```

Comment:
@arjunsinghy96 Also check if the input box is empty. If there is no text in the domain input box, keep this button disabled. We can stop users from submitting empty values here.